src/base/utils/gzip.cpp

```diff
@@ -56,6 +56,7 @@ QByteArray Utils::Gzip::compress(const QByteArray &data, const int level, bool *
     strm.avail_in = uInt(data.size());
     strm.next_out = reinterpret_cast<Bytef *>(tmpBuf.data());
     strm.avail_out = BUFSIZE;
+    strm.total_out = 0;
```

Comment:
I would prefer the initialization was done this way:
```c++
z_stream strm {};
```
But I'm not sure will it cause any unintended side effects, can you try it out?

Reply:
>I would prefer the initialization was done this way instead:
```c++
z_stream strm {};
```

👍 
>But I'm not sure will it cause any unintended side effects, can you try it out?

I don't think it could be worse than an uninitialized value that can actually contain anything.
If zlib expects it to have some predefined initial value it should be documented. IIRC, it is pure "output" field.

Reply:
@glassez It is not documented, but looking on zlib code, it is never initialized, just incremented. 
Since we are not using it, we should not care. However static analysis tools(i.e. Coverity) threat this as uninitialized variable increment.

Reply:
@gxcreator
If you didn't understand, then the meaning of my comment was "initialization with zero in this case certainly can't be worse than a completely uninitialized field".